logos/logos-orchestrator/src/logos/routers/logosnode.py

```
@@ -0,0 +1,514 @@
+                    db.session.rollback()
+                    logger.warning(
+                        "Failed to upsert model profiles for provider %s, the "
+                        "entire row update (base_residency_mb, loaded_vram_mb, "
+                        "kv_budget_mb, measurement_count, last_measured_at) is "
+                        "lost until this recovers",
+                        _resolve_provider_name(provider_id),
+                        exc_info=True,
+                    )
+
+    sample["snapshot_id"] = snapshot_id
+    asyncio.create_task(_logosnode_registry.record_runtime_sample(provider_id, sample))
```

Ruff (0.16.3)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)